addons/account/wizard/accrued_orders.py

```diff
@@ -136,10 +143,7 @@ def _ellipsis(string, size):
             if len(orders) == 1 and self.amount:
                 total_balance = self.amount
                 order_line = order.order_line[0]
```

Comment:
Wtf is this?
So if there are multiple orders, we consider all the lines but not when there is only one?
And we don't care if the first line is a display line or not?

Let's fix this in another commit...

Reply:
Hi @william-andre

Display_type check is done with last commit.
However I'm not sure that we want to change the behaviour of this condition `if len(orders) == 1 and self.amount`.
As I understand the flow, we pass in this condition when having nothing to invoice (unit_price == 0 on each order line) and set the amount manually. As we need product_id to get the receivable/expense accounts, we have to chose arbitrary one line. Or we can decide to set default accounts for this case if you prefer

Reply:
> As I understand the flow, we pass in this condition when having nothing to invoice (unit_price == 0 on each order line)

I don't see where

Reply:
I meant, when quantity == 0
<img width="979" alt="image" src="https://user-images.githubusercontent.com/85162663/191231994-d20413fb-9fb1-493e-9abe-1844ee092b00.png">